packages/yourloops/components/patient-list/current-patient-list/current-patient-list.hook.tsx

```diff
@@ -251,10 +249,7 @@ export const useCurrentPatientListHook = (props: CurrentPatientListProps): Curre
         type: 'number',
         field: PatientListColumns.Variance,
         headerName: t('variance'),
-        renderHeader: () => <CustomHeaderWithTooltip
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏The component `CustomHeaderWithTooltip` looks unused now, it needs to be removed

Reply:
Not resolved + given that the component will be removed, the class `customColumnHeader` should also be removed from `patient-list.styles.ts`